CondTools/RunInfo/interface/BTransitionAnalyzer.h

```diff
@@ -0,0 +1,72 @@
+#ifndef BTRANSITIONANALYZER_H
+#define BTRANSITIONANALYZER_H
+
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Run.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "CondCore/CondDB/interface/Utils.h"
+#include "CondCore/DBOutputService/interface/PoolDBOutputService.h"
+#include "CondFormats/RunInfo/interface/RunInfo.h"
+#include "CondFormats/DataRecord/interface/RunSummaryRcd.h"
+
+namespace cond {
+  template<class T, class R>
+  class BTransitionAnalyzer: public edm::one::EDAnalyzer<edm::one::WatchRuns, edm::one::SharedResources> {
+  public:
+    BTransitionAnalyzer( const edm::ParameterSet& pset ):
+      m_currentThreshold( pset.getUntrackedParameter<double>( "currentThreshold", 18000. ) ) {
+      usesResource( "PoolDBOutputService" );
```

Comment:
@diguida thanks for the clarification!
My other comment does not apply anymore, but I list it here for reference: You write `usesResource("PoolDBOutputService");`, but in general it is better practice to define a static const string in PoolDBOutputService and use this to avoid typos because the compiler checks the name of the variable which is not the case if you use a hard-coded string everytime the resource is used.